pkg/run.go

```diff
@@ -30,6 +30,8 @@ func Run(c *cli.Context) (err error) {
 		l.Fatal(err)
 	}
 
+	os.Setenv("TRIVY_TIMEOUT_SEC", c.String("timeout"))
```

Comment:
You should not use environment variable. It's better to pass it below:
https://github.com/aquasecurity/trivy/blob/master/pkg/scanner/scan.go#L28

Reply:
Add dockerOption argument for ScanImage?

Reply:
Add timeout to `scanOptions`

Reply:
fixed